content/docs/hello-world.md

```diff
@@ -6,7 +6,7 @@ prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+El ejemplo más pequeño de React se ve así:
```

Comment:
```suggestion
El más pequeño de los ejemplos de React se ve así:
```
How it sounds to you?

Reply:
Sounds good @dmoralesm, thanks for the suggestion.